BaragonUI/app/models/Group.coffee

```diff
@@ -76,6 +76,7 @@ class Group extends Model
                 vex.dialog.buttons.NO
             ]
             callback: (data) =>
+                return if data is false
```

Comment:
It already does validation if you try to submit the dialog without filling in the field. The problem was that when the dialog box was dismissed, vex was calling the callback with `false`, and then that was getting sent to the server.